mailer.js

```diff
@@ -276,6 +291,10 @@ module.exports = function (log) {
       'X-Verify-Code': message.code
     }
 
+    if (this.sesConfigurationSet) {
+      headers[X_SES_MESSAGE_TAGS] = sesMessageTagsHeaderValue(templateName)
```

Comment:
So, `.template` is not set on `message`, at least for `verifyEmail`. So I can't consolidate things that way.